docs/glossary.rst

```diff
@@ -0,0 +1,66 @@
+===============================================================================
+XBRL Glossary
+===============================================================================
+
+Axis
+^^^^
+
+Axes are defined in the Taxonomy, and are used as a part of the
```

Comment:
Need to explain how the Axes relate to primary keys in the database tables that end up getting produced. An example would be helpful.